packages/wrapped-keys/src/lib/api/export-private-key.ts

```diff
@@ -0,0 +1,55 @@
+import { decryptToString } from '@lit-protocol/encryption';
+
+import { CHAIN_ETHEREUM, LIT_PREFIX } from '../constants';
+import { fetchPrivateKeyMetadata } from '../service-client';
+import { ExportPrivateKeyParams, ExportPrivateKeyResult } from '../types';
+import {
+  getFirstSessionSig,
+  getPkpAccessControlCondition,
+  getPkpAddressFromSessionSig,
+} from '../utils';
+
+/** Exports a previously persisted private key from the wrapped keys service for direct use by the caller, along with the keys metadata
+ *
+ * @param { ExportPrivateKeyParams } params Parameters required to export the private key
+ */
+export async function exportPrivateKey(
+  params: ExportPrivateKeyParams
+): Promise<ExportPrivateKeyResult> {
+  const { pkpSessionSigs, litNodeClient } = params;
+
+  const sessionSig = getFirstSessionSig(pkpSessionSigs);
+  const pkpAddress = getPkpAddressFromSessionSig(sessionSig);
+  const allowPkpAddressToDecrypt = getPkpAccessControlCondition(pkpAddress);
+
+  const privateKeyMetadata = await fetchPrivateKeyMetadata({
+    sessionSig,
+    litNetwork: litNodeClient.config.litNetwork,
+  });
+
+  const { ciphertext, dataToEncryptHash, ...privateKeyMetadataMinusEncrypted } =
+    privateKeyMetadata;
+
+  const decryptedPrivateKey = await decryptToString(
+    {
+      accessControlConditions: [allowPkpAddressToDecrypt],
+      chain: CHAIN_ETHEREUM,
+      ciphertext,
+      dataToEncryptHash,
+      sessionSigs: pkpSessionSigs,
+    },
+    litNodeClient
+  );
+
+  // It will be of the form lit_<privateKey>
+  if (!decryptedPrivateKey.startsWith(LIT_PREFIX)) {
+    throw new Error(
+      `PKey was not encrypted with salt; all wrapped keys must be prefixed with '${LIT_PREFIX}'`
```

Comment:
Did we conclude if we want to allow non salted keys to resolve? Since it is technically possible to upload one without the salt can we throw here?

Reply:
They technically can do that but only in the Custom Wrapped Keys so we can only add it the docs informing them